django_project/lesson/views/question.py

```diff
@@ -126,8 +125,7 @@ def get_success_url(self):
         })
 
 
-class QuestionOrderView(
-    StaffuserRequiredMixin, QuestionMixin, ListView):
+class QuestionOrderView(LoginRequiredMixin, QuestionMixin, ListView):
     """List view to order questions"""
```

Comment:
The code looks okay,
Lets add a dot **.** to end the docstring

Reply:
I will fix them in one go after this PR with a regex ;-)
Thanks